OPENPGP PLUGIN: Show key as expired or revoked in account manager window

eerielili opened the pull request:
A small update concerning the openpgp plugin: with this, the key can be shown either as expired or revoked in the account manager windows concerning an account.
![image](https://github.com/mxlgv/dino/assets/81772782/dfe52142-061f-462e-9391-a8bace9306ff)

eerielili committed:
OPENPGP PLUGIN: Show key as expired or revoked in account manager window

	- updated template translation file for the plugin.
	- updated French translation for the plugin as well.
        - Update German and Russian .po files skeletal as well.

mxlgv commented:
@giantplaceholder do we need this change?

mxlgv commented on plugins/openpgp/src/account_settings_entry.vala:
```diff
@@ -69,7 +69,7 @@ public class AccountSettingsEntry : Plugins.AccountSettingsEntry {
         }
         if (keys.size == 0) {
-            label.set_markup(build_markup_string(_("Key publishing disabled"), _("No keys available. Generate one!")));
+            label.set_markup(build_markup_string(_("Key publishing disabled"), _("No keys available. Generate one or check if your keys aren't expired or revoked!")));
```
You also need to fix the gettext files, at least for Russian, English and German.

eerielili replied:
Okay. I'll update the dino.pot file and fr.po file at least, I don't know German and Russian that much.

eerielili commented:
@giantplaceholder Thanks a lot for the review. I updated the translation template `plugins/openpgp/po/dino-openpgp.pot` just now, also added some translations for my language as well.

mxlgv commented:
@eerielili please rebase CI for Windows has been fixed.

eerielili commented:
@mxlgv done

mxlgv commented on plugins/openpgp/src/account_settings_entry.vala:
```diff
@@ -88,6 +88,18 @@ public class AccountSettingsEntry : Plugins.AccountSettingsEntry {
         set_label_active(selected);
 
         combobox.changed.connect(key_changed);
+        if (account_key != null){
```
missing backspace `) {`

mxlgv commented on plugins/openpgp/po/ru.po:
```diff
@@ -17,38 +18,62 @@ msgstr ""
 msgid "Encryption"
 msgstr "Шифрование"
+
+#~ msgid "No keys available. Generate one!"
```
Remove completely if the code is not used

mxlgv commented on plugins/openpgp/po/fr.po:
```diff
@@ -5,49 +5,73 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
```
Why was `PACKAGE VERSION` removed?

mxlgv commented on plugins/openpgp/po/fr.po:
```diff
@@ -5,49 +5,73 @@
 msgid "Encryption"
 msgstr "Chiffrement"
+
+#~ msgid "No keys available. Generate one!"
```
Remove completely if the code is not used

mxlgv commented on plugins/openpgp/po/ru.po:
```diff
@@ -5,10 +5,11 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
```
Why was `PACKAGE VERSION` removed?

mxlgv commented on plugins/openpgp/po/ru.po:
```diff
@@ -17,38 +18,62 @@ msgstr ""
+msgid ""
+"No keys available. Generate one or check if your keys aren't expired or "
+"revoked!"
+msgstr ""
```
Please add this translation:
```
"Нет доступных ключей. Сгенерируйте их или проверьте срок действия."
"Также возможно, что ключ был отозван!"
```

mxlgv commented on plugins/openpgp/po/ru.po:
```diff
@@ -17,38 +18,62 @@ msgstr ""
+
+#: account_settings_entry.vala:95
+msgid "expired!"
+msgstr ""
```
Please add this translation:
```
"истёк!"
```

mxlgv replied:
?

mxlgv replied:
@eerielili can you see this?  This is my language.  I offered a fairly accurate translation for each.

mxlgv commented on plugins/openpgp/po/ru.po:
```diff
@@ -17,38 +18,62 @@ msgstr ""
+
+#: account_settings_entry.vala:95
+msgid "revoked!"
+msgstr ""
```
Please add this translation:
```
"отозван!"
```

mxlgv replied:
?

mxlgv commented on plugins/openpgp/po/ru.po:
```diff
@@ -17,38 +18,62 @@ msgstr ""
+#: account_settings_entry.vala:96
+#, c-format
+msgid "Your key %s is %s"
+msgstr ""
```
Please add this translation:
```
"Ваш ключ %s %s"
```

mxlgv replied:
?

mxlgv commented on plugins/openpgp/po/de.po:
```diff
@@ -2,50 +2,72 @@ msgid ""
+msgid ""
+"No keys available. Generate one or check if your keys aren't expired or "
+"revoked!"
+msgstr ""
```
Please add this translation:
```
"Keine Schlüssel vorhanden. Erstellen Sie einen oder prüfen Sie,"
"ob Ihre Schlüssel nicht abgelaufen sind oder widerrufen!"
```

mxlgv commented on plugins/openpgp/po/de.po:
```diff
@@ -2,50 +2,72 @@ msgid ""
+
+#: account_settings_entry.vala:95
+msgid "revoked!"
+msgstr ""
```
Please add this translation:
```
"widerrufen!"
```

mxlgv commented on plugins/openpgp/po/de.po:
```diff
@@ -2,50 +2,72 @@ msgid ""
+
+#: account_settings_entry.vala:95
+msgid "expired!"
+msgstr ""
```
Please add this translation:
```
"abgelaufen!"
```

mxlgv commented on plugins/openpgp/po/de.po:
```diff
@@ -2,50 +2,72 @@ msgid ""
 msgid "Encryption"
 msgstr "Verschlüsselung"
 
+#~ msgid "No keys available. Generate one!"
```
Remove completely if the code is not used

mxlgv commented on plugins/openpgp/po/de.po:
```diff
@@ -2,50 +2,72 @@ msgid ""
+#: account_settings_entry.vala:96
+#, c-format
+msgid "Your key %s is %s"
+msgstr ""
```
Please add this translation:
```
"Ihr Schlüssel %s ist %s"
```

eerielili commented:
Thanks. Wait a minute, I must pust a russian translation i deleted by mistake

eerielili commented:
Updated the Russian translation. I used deepl.com to fill out the gaps since it's not my native language.

mxlgv commented:
@eerielili  Thank you for your contribution to the project.     After the merger I will correct the translation.  Don't worry about translation)

eerielili reviewed (commented):
done

eerielili commented:
@mxlgv
I did update my branch to fit the new account settings UI.
